src/support/components/PayGSupportOverlay.tsx

```diff
@@ -96,20 +100,34 @@ const PayGSupportOverlay: FC<OwnProps> = () => {
   }
 
   const handleSubmit = async () => {
-    const email = quartzMe?.email
-    const descriptionWithOrgId = `${description}  [Org Id: ${orgID}]`
+    let userID: string, userEmail: string, orgName: string, orgID: string
+
+    if (shouldUseQuartzIdentity()) {
+      userID = identityUser.id
+      userEmail = identityUser.email
+      orgName = identityOrg.name
+      orgID = identityOrg.id
+    } else {
+      // Optional chaining operator needed because quartzMe's initial reducer state is null.
+      userID = quartzMe?.id
+      userEmail = quartzMe?.email
+      orgName = quartzOrg?.name
+      orgID = quartzOrg?.id
+    }
```

Comment:
Can you think of a way that this logic can be encapsulated so that we don't need to sprinkle it all over the codebase? Or is this maybe just a one-off situation?

Reply:
The clean way to do would be to just ignore identity and use the `quartzMe` state, since even when `identity` is on, `convertIdentityToMe` converts the identity shape into the shape of `quartzMe` so that it can be consumed throughout the app without lots of `if` blocks.

Unfortunately, we can't do that here because the _org id_ isn't returned by the `quartzMe` endpoint. So I think this is a brief 'one-off', especially since `quartzIdentity` is in prod. So this might be one of the last times we have to do it before the ff is off.

Reply:
Alternatively - we could just leave it using `quartzMe` and the idpe `org` endpoint for now. And worry about using identity later. That's less code.